cfme/configure/configuration/__init__.py

```diff
@@ -178,10 +178,8 @@ def server_zone_description():
 
 
 def server_region_string():
-    if store.current_appliance.is_downstream:
-        return "CFME Region: Region {} [{}]".format(*server_region_pair())
-    else:
-        return "ManageIQ Region: Region {} [{}]".format(*server_region_pair())
+    return "{} Region: Region {} [{}]".format(
```

Comment:
This code will work after 5.7.0.4, so we'll just leave this PR open till the next build